packages/nc-gui/components/general/Overlay.vue

```diff
@@ -11,6 +11,7 @@ interface Props {
   target?: TeleportProps['to']
   teleportDisabled?: TeleportProps['disabled']
   transition?: boolean
+  lightBackground?: boolean
```

Comment:
You've added `lightBackground` in interface but didn't add it in L23 (defineProps). It may get `Unresolved variable or type`.